lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1607,7 +1640,8 @@ spv_ostream &operator<<(spv_ostream &O, SPIRVModule &M) {
     << MI.ForwardPointerVec
     << TopologicalSort(MI.TypeVec, MI.ConstVec, MI.VariableVec,
                        MI.ForwardPointerVec)
-    << SPIRVNL() << MI.DebugInstVec << SPIRVNL() << MI.FuncVec;
+    << SPIRVNL() << MI.AsmVec << SPIRVNL() << MI.DebugInstVec << SPIRVNL()
```

Comment:
I think this should be done only if corresponding extension is enabled

Reply:
Done